readme.txt

```diff
@@ -25,6 +25,9 @@ Column separator for Beaver Builder. Separate Beaver Builder columns with differ
 
 == Changelog ==
 
+= 1.0.2 =
```

Comment:
version number in readme header is not updated

Reply:
Updated